doc/source/conf.py

```diff
@@ -68,6 +68,17 @@
         '../examples/applications',
         '../examples/developers',
     ]),
+    'binder': {
+        # Required keys
+        'org': 'scikit-image',
+        'repo': 'docs',
+        'branch': 'gh-pages',  # Can be any branch, tag, or commit hash
+        'binderhub_url': 'https://mybinder.org',  # Any URL of a binderhub.
+        'dependencies': '../../requirements/binder/requirements.txt',
```

Comment:
why not `requirements/binder.txt`?

Reply:
because this file will be copied directly by sphinx-gallery without renaming it and binder will need it to be called `requirements.txt`.